lib/neuron/connection.ex

```diff
@@ -1,11 +1,35 @@
 defmodule Neuron.Connection do
+  alias Neuron.Config
   @moduledoc false
 
-  def post(url, query, headers \\ []) do
+  def post(url, _) when is_nil(url) do
+    raise ArgumentError, message: "you need to supply an url"
+  end 
+
+  def post(url, query) do
     HTTPoison.post(
       url,
       query,
-      Keyword.merge(["Content-Type": "application/graphql"], headers)
+      %{},
```

Comment:
If I'm not missing anything, this argument shouldn't be here, right?
https://hexdocs.pm/httpoison/HTTPoison.html#post/4